nservicebus/azure-storage-queues/configuration.md

```diff
@@ -62,6 +62,13 @@ Defaults:
  * 32 in Version 7
 
 
+#### SerializeMessageWrapperWith
+
+Azure Storage Queues Transport wraps messages in a transport specific message containing message metadata. By default, Azure Storage Queues Transport uses the same serializer for the message wrapper as configured for the contained message. While NserviceBus Version 6 supports [multiple deserializers](/nservicebus/serialization#specifying-additional-deserializers) for messages, all endpoints in the same system needs to use the same serializer for the wrapper message. In NServiceBus Version 6 and above, it is possible to configure a different wrapper serializer by using the `SerializeMessageWrapperWith` option.
```

Comment:
> Can you add more detail? what metadata?

is this relevant?

>  Is it version difference between NServiceBus Version 5 and 6? Because it feels like it conflicts with the below:

the situation is:
* nsb 5 uses the serializer configured by UseSerializer for the wrapper message. Since we don't support multi-deserializers in v5, it was required to use the same serializer across all endpoints anyway.
* nsb 6 supports multi-deserializers. This support is limited to the message (contained in the wrapper) itself. the wrapper still needs to be serialized using the same serializer across all endpoints.

would a description like this be more clear?